src/netfx.force.conflict/src/TypeForwards.471.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+// System.Diagnostics.Tracing
```

Comment:
Why are you scoping to just these? We need to make sure that we we introduce conflicts for anything that doesn't match in the platform. If this is the only set then fine.

Reply:
These are the only 4 that exist in TypeForwards.cs file and also are part of the 12 shims that we need to binplace with the app on 471. That said, I'm not entirely sure why we only had 4 of those 12 in the original TypeForwards.cs, and I'm also not sure if just includding these could be a problem in the future in case we ship a newer version of the OOB packages.

Reply:
The original idea is that we would include references to all the assemblies that are shipped in the support package to ensure that the correct binding redirects get generated. I would error on the side of having representatives for all the libraries we have in the support package.

Reply:
I would include one member from each of the 12 binaries we have in the bin folder to error on the side of caution.

Reply:
Done